logger.go

```diff
@@ -34,15 +33,10 @@ type Shuttle struct {
 
 // NewShuttle creates a shuttle client from a Config.
 func NewShuttle(cfg *LoggerConfig) (*Shuttle, error) {
-	token, ok := cfg.URL.User.Password()
-	if !ok {
-		return nil, errors.New("no logplex password provided")
-	}
-
 	// Create shuttle config.
 	conf := shuttle.NewConfig()
 	conf.LogsURL = cfg.URL.String()
-	conf.Appname = token
+	conf.Appname = "heroku"
```

Comment:
I'm assuming `"heroku"` is the right value here for all data services. You never log as the `"app"`, right?